docs/API.md

```diff
@@ -1037,13 +1037,13 @@ An object with the following properties:
 The value of `"test_params"` is an object with the following properties:
 
 * `"client_id"`: A *client id*, optional. (default: unset)
-* `"profile"`: A *profile name*, optional (default `"default"`). Run the tests using the given profile.
-* `"config"`: **Deprecated.** A string, optional. Ignored. Specify profile instead.
+* `"profile"`: A *profile name*, optional (default: `"default"`). Run the tests using the given profile.
+* `"config"`: **No longer supported**. Use `"profile"` instead.
```

Comment:
In this case I think we can just remove it.

Reply:
Removed in 52dfb17.